sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1989,6 +1996,8 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val isTestEnabled: Boolean = get(TEST_CONF)
 
+  lazy val testRetryOOMInjectionEnabled : Boolean = get(TEST_RETRY_OOM_INJECTION_ENABLED)
```

Comment:
Currently, `spark.rapids.sql.test.enabled` is being used in the CI as false if we don't allow CPU execs, e.g. if everything should be on the GPU `spark.rapids.sql.test.enabled=false`. Is that the flag you had in mind? If so maybe we need to introduce a config that is "no really, this is running under tests"

Reply:
ok thanks. I'll leave it as is for now since this is a nit, I actually am interested in injecting OOMs even for these, but I do agree these are a small subset of the tests.